### PR TITLE
Update documentation for version 0.2.0

### PR DESCRIPTION
**Pull request by itziakos**

### CODE REVIEW

**Comment by itziakos** [comment]
@dpinte and @nathanfranklin do you have time to review the documentation of the new version of the simphony_mayavi plugin?

**Comment by nathanfranklin** [comment]
@itziakos , I can take a look before lunch tomorrow.

**Comment by itziakos** [comment]
Thanks @nathanfranklin, tomorrow is perfect

File: doc/source/mayavi2.rst [hunk 1]
```diff
@@ -1,11 +1,20 @@
 Mayavi2
 =======
 
-The Simphony-Mayavi library provides a set of tools to easily create
-mayavi ``Source`` instances from SimPhoNy CUDS containers. With the
-provided tools one can use the SimPhoNy libraries to work inside the
-Mayavi2 application, as it is demonstrated in the examples.
+The Simphony-Mayavi library provides a plugin for Mayavi2 easily create
```
Comment (nathanfranklin) [inline doc/source/mayavi2.rst]: "to easily"

**Comment by itziakos** [comment]
The documentation can be found in http://simphony-mayavi.readthedocs.org/en/update-documentation/index.html

File: README.rst [hunk 2]
```diff
@@ -85,12 +85,12 @@ After installation the user should be able to import the ``mayavi`` visualizatio
 Directory structure
 -------------------
 
-There are four subpackages:
```
Comment (nathanfranklin) [inline README.rst:88]: visualisation with a ("z") in example above (in line 84),  "from simphony.visualization import mayavi_tools"

File: doc/source/simphony.rst [hunk 3]
```diff
@@ -35,4 +35,57 @@ allows the common `mayavi operations`_.
...
+
+.. literalinclude:: ../../examples/particles_vtk_example.py
+
+Adapting VTK datasets
+---------------------
+
+The :func:`~simphony_mayavi.adapt2cuds.adapt2cuds` function is
+available to wrap common VTK datsets into top level CUDS
+containers. The function will attempt to automatically adapt the
+(t)vtk Dataset into a CUDS container. But when this fails the user can
```
Comment (nathanfranklin) [inline doc/source/simphony.rst]: Suggestion:  " But when this fails, the user..." ->  "If the function fails in determining  the type of CUDS container, the user.."

File: doc/source/simphony.rst [hunk 4]
```diff
@@ -35,4 +35,57 @@ allows the common `mayavi operations`_.
...
+always force the kind of the container and mapping of the included
+attribute data into corresponding CUBA keys.
```
Comment (nathanfranklin) [inline doc/source/simphony.rst]: `and mapping of the included attribute data into corresponding CUBA keys` 

this statement about the mapping of CUBA/attribute data is independent of the function automatically finding the CUDS container type or user  setting the 'type' parameter, right?    If I understood that correctly, then i would suggest putting it in another sentence like:  `The user can (should?) also set the mapping of the included attribute data into corresponding CUBA keys`

**Comment by nathanfranklin** [comment]
@itziakos ,  great stuff!  i ran the examples (very clear and comprehensive) and read the documentation.  only found some minor things.

**Comment by nathanfranklin** [comment]
I did have to install `envisage` after getting an ImportError when running one of the examples.

**Comment by itziakos** [comment]
Thanks @nathanfranklin.

> I did have to install envisage after getting an ImportError when running one of the examples.

it was a bud requirement in setup.py. Good catch

**Pull request merged**